Source/WebKit/UIProcess/Extensions/WebExtension.cpp

```diff
@@ -2220,6 +2227,124 @@ void WebExtension::populateCommandsIfNeeded()
     }
 }
 
+std::optional<WebExtension::DeclarativeNetRequestRulesetData> WebExtension::parseDeclarativeNetRequestRulesetObject(const JSON::Object& rulesetObject, RefPtr<API::Error>& error)
+{
+    auto rulesetID = rulesetObject.getString(declarativeNetRequestRulesetIDManifestKey);
+    if (rulesetID.isEmpty()) {
+        error = createError(Error::InvalidDeclarativeNetRequest, WEB_UI_STRING("Empty `declarative_net_request` ruleset id.", "WKWebExtensionErrorInvalidDeclarativeNetRequestEntry description for empty ruleset id"));
```

Comment:
The original should have been this way:
```suggestion
        error = createError(Error::InvalidDeclarativeNetRequest, WEB_UI_STRING("Empty or invalid `id` in `declarative_net_request` manifest entry.", "WKWebExtensionErrorInvalidDeclarativeNetRequestEntry description for empty or invalid id in declarative_net_request manifest entry"));
```